Added Login page

prahe184 opened the pull request:
[#Description:](url)
![Screenshot 2025-04-22 155116](https://github.com/user-attachments/assets/77d342de-3ffa-46c3-8689-266249ea0693)

Created a login page using Typescript + React.
For styling, strictly followed the theme which is same as Homepage.
Used Tailwind CSS.

Closes #11

CSE-TechClub commented:
good job @prahe184 .

But we also want the Register page , which consist of ..
- email 
- name
- college USN (University Serial Number)
- password 

And need to change the some things in login ...
- As it is Club website we need ask USN and password  .. 

So kindly change Email into  USN (String <type>)  in login page and Create Register page with link to login page

prahe184 commented:
@CSE-TechClub , I've added Register page too and changed email into USN in Login page. 

https://github.com/user-attachments/assets/320d2417-0701-44ea-aa3d-cf0c2f1183bb

 I've added Register page too and changed email into USN in Login page

CSE-TechClub commented:
Respected @prahe184 ma'am , 

You have done a Great Job  , Thank you for contributing our website .You are always welcome to Contribute .

Students Club .
